$NetBSD: patch-BUILD-NetBSD,v 1.1 2018/09/27 20:02:50 gdt Exp $

Export RELEASEMACHINEDIR, and default it to $arch, so that using
evbearmv7hf-el results in that being the subdir in releasedir, so that
6/7 don't overwrite each other.

Upstream has been notified, but perhaps this belongs in in build.sh.

--- BUILD-NetBSD.orig	2016-05-21 19:35:53.000000000 +0000
+++ BUILD-NetBSD
@@ -193,6 +193,9 @@ fi
 
 # Parse fixed arguments.
 cmd=$1
+# Note that arch will typically be a "uname -m" value, in simple
+# cases, or an alias that defines a combination of uname -m and -p,
+# like evbearmv7hf-el which maps to evbarm and earmv7hf.
 arch=$2
 
 # Make first-cut decision about UNPRIV (UNPRIV if non-root)
@@ -248,9 +251,12 @@ export OBJDIR="$BUILDBASE"/"$arch"
 # Place all destdirs together, separated by architecture.
 export DESTDIR="$BUILDBASE"/destdir/"$arch"
 
-# Releasedir has $arch subdirs by default, so omit implied "/$arch".
+# Note that RELEASEDIR is the parent directory, in which there is /$arch.
 export RELEASEDIR="$BUILDBASE"/releasedir
 
+# build.sh uses $arch by default
+export RELEASEMACHINEDIR=$arch
+
 if [ "$cmd" = "" ]; then
     cmd=all
     log "Command omitted - assuming \"all\""
